source/whats-new.txt

```diff
@@ -88,6 +88,12 @@ New features of the 4.8 driver release include:
   or ``TreeSet``.
 - Support for recursive record types, records whose type
   definitions contain cycles.
+- Inclusion of the ``disambiguatedPaths`` field in change stream update events
+  when the ``showExpandedEvents`` server feature is enabled on MongoDB v6.1
+  and later. Learn more about this field from the `getDisambiguatedPaths() <https://mongodb.github.io/mongo-java-driver/4.8/apidocs/mongodb-driver-core/com/mongodb/client/model/changestream/UpdateDescription.html#getDisambiguatedPaths()>`__
+  API documentation. Learn more about the ``showExpandedEvents`` setting from
+  the :manual:`Expanded Events </reference/change-events/#expanded-events>` 
+  Server manual page.
```

Comment:
S: [Dave Cuthbert and others suggest](https://mongodb.slack.com/archives/C0339UHK56H/p1676061494356189) we use "server documentation" or "Server Documentation", depending on context.

```suggestion
  page in the server documentation.
```

Reply:
I'm fine with changing this instance, but was that suggestion accepted team-wide or was it a personal preference? I probably would want to contribute my own input in a wider discussion such as a style guide for a team-wide standard.